README.md

```diff
@@ -8,6 +8,29 @@ This module allows to test the performance and response time of individual endpo
 ## Installation
 
 ```
-composer require spryker-sdk/benchmark
+composer require --dev spryker-sdk/benchmark
 ```
 
+
+## Usage
+
+### How to use in Spryker projects
+Make sure you include the benchmark as `require-dev` dependency:
+```
+composer require --dev spryker-sdk/benchmark
```

Comment:
why twice? it seems you already have the command above.

Reply:
done